app/controllers/thredded/autocomplete_users_controller.rb

```diff
@@ -6,17 +6,27 @@ class AutocompleteUsersController < Thredded::ApplicationController
     def index
       authorize_creating PrivateTopicForm.new(user: thredded_current_user).private_topic
       users = params.key?(:q) ? users_by_prefix : users_by_ids
+
       render json: {
-        results: users.map do |user|
-          { id:         user.id,
-            name:       user.send(Thredded.user_name_column),
-            avatar_url: Thredded.avatar_url.call(user) }
-        end
+          results: users.map do |user|
+            { id:         user.id,
+              name:       display_name(user),
+              avatar_url: Thredded.avatar_url.call(user) }
+          end
       }
     end
 
     private
 
+    # Defines how the name is displayed in the autocomplete search results
+    def display_name(user)
+      if Thredded.user_display_name_method == :to_s
+        user.send(Thredded.user_name_column)
+      else
+        "#{user.send(Thredded.user_name_column)} - #{user.send(Thredded.user_display_name_method)}"
+      end
+    end
+
```

Comment:
I do not think it would be rare at all if the display name is different from the user name column. Are you suggesting I just override this locally and forget about the pull request entirely?